pom.xml

```diff
@@ -27,6 +27,9 @@
     <distro.openmrscoreDir>${distro.baseDir}/openmrs_core</distro.openmrscoreDir>
     <distro.openmrsmodulesDir>${distro.baseDir}/openmrs_modules</distro.openmrsmodulesDir>
     <distro.bahmnireportsDir>${distro.baseDir}/bahmni_reports</distro.bahmnireportsDir>
+    <distro.openerpatomfeedserviceDir>${distro.baseDir}/openerp_atomfeed_service</distro.openerpatomfeedserviceDir>
+    <db-event.destinations>direct:odoo-event-listener</db-event.destinations>
```

Comment:
Could you make sure to separate those in their own block with the right title (as comment).